### DISCUSSION TITLE
setup is unable to acces the manager

### DISCUSSION THREAD

**Discussion by mohasmr** [Security-Onion-Solutions/securityonion#12478] [2.4]
### Discussed in https://github.com/Security-Onion-Solutions/securityonion/discussions/12465

<div type='discussions-op-text'>

<sup>Originally posted by **mohasmr** February 29, 2024</sup>
i have a problem i cant even identify the causes when i install a desktop to linked to a distributed manager and try to join it to an existing grid i have problem of connectivity 
manager ip:192.168.10.10
desktop ip : 192.168.10.90 
they are on the same network , in the manager console i puted the right configuration
![Capture](https://github.com/Security-Onion-Solutions/securityonion/assets/135219197/83bcb0c1-fdf3-4803-bf84-99e7d8b4eca3)
even i try with **so-firewall-minion --role=DESKTOP --ip=192.168.10.90**  there is no changes
i even tryied to shutdow the firewalld and droped all iptables but nothing 
the two security onion(desktop and manager) cant even ping each others 
please to help im stuck :'(

</div>

**Comment by kevinelwell** [comment]
I have the same challenge. Unable to join the analyst desktop to grid.

**Reply by kevinelwell** [reply]
I was able to solve my challenge. When prompted to join the desktop vm to the grid, I selected yes. It failed. I ran the command "so-firewall-minion --role=DESKTOP --ip=192.168.1.10" on the manager vm. The command ran successfully. I then went back to the desktop vm and selected "try again" and it ran successfully. I then was able to see the desktop vm in the grid.

**Reply by mohasmr** [reply]
is your manager installed like a distributed manager or a standalone ??

**Reply by kevinelwell** [reply]
Standalone

**Comment by InfosecGoon** [comment]
Okay, just to make sure that I'm answering the right question - you're trying to join a Security Onion Desktop to an existing Security Onion grid? How did you install the Desktop?

**Reply by mohasmr** [reply]
I got 2 different pcs and security onion desktop is installed and the manager is installed on the other one ,every security onion is on bridged mode and is connected on the same Ethernet cable , all the virtual cards and the ethernet are on the same ip range which is 192.168.10.0/24
In the installation of security onion desktop i have chosen to join it to an existing grid which is SO MANAGER
I noticed that the 2 SO VMS cannot see or even ping on each other

**Reply by mohasmr** [reply]
![error](https://github.com/Security-Onion-Solutions/securityonion/assets/135219197/5d01a665-7e1f-47fc-889e-ad0738d462d5)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/135219197/b5778274-54b9-4ef3-9c8a-085749db457a)

**Comment by mohasmr** [comment]
the issues caused by a missconfiguration on the switch the  so-firewall-minion --role=DESKTOP --ip=192.168.10.90 working correctly
